package.json

```diff
@@ -25,20 +25,21 @@
     "postinstall": "npm run tsd-update && tsd install && tsd link",
     "start": "rimraf build && webpack --watch",
     "test": "webpack --config webpack.test.config.js && browserify build/test.js | tape-run | tap-spec",
-    "prepack": "npm test", 
+    "prepack": "npm test",
     "pack": "./crxmake.sh"
   },
   "dependencies": {
-    "rxjs": "^5.0.0-beta.0",
     "angular2": "^2.0.0-beta.0",
     "basscss": "^7.0.4",
     "browserify": "^12.0.1",
     "crypto": "0.0.3",
     "es6-promise": "^3.0.2",
     "es6-shim": "^0.33.6",
+    "file-loader": "^0.8.5",
```

Comment:
Could you please move this under devDependencies since it's only needed for webpack?